sheetsync/sheetsync/sheets.py

```diff
@@ -294,7 +298,11 @@ def __init__(self, client, sheet_id, worksheets, bustime_start, edit_url, alloca
 
 	def get_rows(self):
 		# only need to update the shifts once per sync
-		self.latest_shifts = common.shifts.parse_shifts(self.shifts)
+		try:
+			self.latest_shifts = common.shifts.parse_shifts(self.shifts)
+		except Exception as e:
+			logging.error('Error parsing shifts with {}. Using previous shifts definition.'.format(e))
```

Comment:
nitpick: Instead of printing the error explicitly, use `logging.exception` which automatically includes the error and traceback.
```suggestion
		except Exception:
			logging.exception('Error parsing shifts. Using previous shifts definition.')
```